serving/samples/helloworld-scala/README.md

```diff
@@ -0,0 +1,107 @@
+# Hello World - Scala using Akka HTTP sample
+
+A microservice which demonstrates how to get set up and running with Knative Serving when using [Scala](https://scala-lang.org/) and [Akka](https://akka.io/) [HTTP](https://doc.akka.io/docs/akka-http/current/). It will respond to a HTTP request with a text specified as an `ENV` variable named `MESSAGE`, defaulting to `"Hello World!"`.
+
+##Prerequisites
+
+* A Kubernetes cluster [installation](https://github.com/knative/docs/blob/master/install/README.md) with Knative Serving up and running.
+* [Docker](https://www.docker.com) installed locally, and running, optionally a Docker Hub account configured or some other Docker Repository installed locally.
+* [Java JDK8 or later](https://adoptopenjdk.net/installation.html) installed locally.
+* [Scala's](https://scala-lang.org/) standard build tool [sbt](https://www.scala-sbt.org/) installed locally.
+
+## Configuring the sbt build
+
+Have a look at the [build.sbt](build.sbt) and **specify your Docker repository name**:
+
+```
+// If you want to use your Docker Hub repository, set this to "docker.io/yourusername/yourreponame".
+// If you use Minikube and want to use the Docker Repository inside Minikube, either set this to "dev.local" or
+// if you want to use another repository name, then you need to run the following command after `docker:publishLocal`:
+// `docker tag yourreponame/helloworld-scala:<version> dev.local/helloworld-scala:<version>`
+// Otherwise Knative Serving won't be able to resolve this image from the Minikube Docker Repository, also do not forget: `eval $(minikube docker-env)` before publishing in that case.
+dockerRepository := Some("your_repository_name")
+```
+
+You can learn more about the build configuration syntax [here](https://www.scala-sbt.org/1.x/docs/Basic-Def.html).
+
+## Configuring the Service descriptor
+
+Importantly, in [helloworld-scala.yaml](helloworld-scala.yaml) **change the image reference to match up with the repository**, name, and version specified in the [build.sbt](build.sbt) in the previous section.
+
+```
+image: "your_repo_name/helloworld-scala:0.0.1"
```

Comment:
I think our current docs style is to present the full yaml in-line for better readability. At some point, we may be able to perform includes once the website is up and running, but just copy-paste for now (unless Sam or another tech writer corrects me).

Reply:
That makes sense, I'll copy-paste the service descriptor yaml